core/templates/domain/story/StoryContentsObjectFactory.ts

```diff
@@ -192,7 +196,7 @@ export class StoryContents {
     }
     if (nodeId === this._initialNodeId) {
       if (this._nodes.length === 1) {
-        this._initialNodeId = null;
+        this._initialNodeId = '';
```

Comment:
Why is this changed to `''`?

Reply:
We cannot assign null value to type `string` and cannot change type of `_initialNodeId` to `string | null` either

Reply:
Why?

Reply:
We would have to change the return type of function `getInitialNodeId` to `string | null` then ,this can break something up in `story-update.service.ts` where it is used.

Reply:
Quite the opposite, if you change it to `''` you need to verify that in _story-update.service.ts_ we don't actually break anything (imagine if we checked for `null` explicitly there). If you change the typing only the types can break.

Reply:
done